### ISSUE TITLE
Using flickity with bootstrap 4 for horizontal cards "scrolling" (responsive)

### ISSUE THREAD

**Issue by KevinGagnon** [metafizzy/flickity#742]
Is flickity supposed to be compatible with BS4? If so, could anyone provide an example on the correct way to integrate it? 

I tried different way : one BS4 container div specifically for flickity; using BS4 classes for row/col. I ended up having different issues. For example cards would disappear or they would not be the same size. And changing the screen size was pretty bad too :/

In this particular case, I would like to display a certain number of cards at a time on a large screen and only one at a time on a phone screen.

**Comment by jmharveys** [comment]
Yes, on this site : https://foodlavie.com/

:)

**Comment by KevinGagnon** [comment]
@jmharveys This is the result I am looking for, thanks for showing me it is possible! I did get rid of the bootstrap cards and used nested DIVs only (like on your site). Works pretty well now and it is responsive inside a BS4 container-fluid div. I was even able to make it coexist with a BS4 carousel. CSS was my main issue I believe. Once I copied the CSS from an example on http://flickity.metafizzy.co/ everything was fixed. Thanks again :)
